### ISSUE TITLE
Could the readme be improved to include examples of passing options to plugins? 

### ISSUE THREAD

**Issue by kevinv92** [sindresorhus/gulp-imagemin#248]
Maybe it's just me but I thought you pass options to gulp-imagemin and it just passes it down to the plugins. 

I have been trying to do this and was wondering why I'm not getting the output that I want
```javascript
...
 .pipe(imagemin({progressive: true}))
...
```

but after some googling around, you actually pass them directly to the plugins

```javascript
...
 .pipe(imagemin([imagemin.jpegtran({ progressive: true })], {verbose: true})
...
```
The latter approach worked and I've got output that I expected. I don't know why I assumed that there's such a feature like the first approach I've done.

**Comment by jordanbrauer** [comment]
It's not just you. I left this plugin behind a while back for `gulp-image` because of this exact reason - lack of documentation as well as not fully optimizing the images, however that is an issue for a different post lol.

I second @kevinv92 motion for improved docs.

**Comment by sindresorhus** [comment]
Fixed by #252.
